src/core/components/form/Select.vue

```diff
@@ -69,11 +75,17 @@ export default {
       this.$emit('input', val);
       this.$refs.selectInput.blur();
     },
+    formatStringForFiltering (str) {
+      return escapeRegExp(removeDiacritics(str.toLowerCase()));
+    },
     onFilter (val, update) {
       update(() => {
         if (val) {
-          const value = val.toLowerCase();
-          this.innerOptions = this.options.filter(opt => opt.label.toLowerCase().includes(value));
+          const formattedValue = this.formatStringForFiltering(val);
+          this.innerOptions = this.options.filter((opt) => {
+            const formattedOptionLabel = this.formatStringForFiltering(opt.label);
+            return formattedOptionLabel.match(new RegExp(formattedValue, 'i'));
```

Comment:
je crois que tu peux mettre juste `return this.formatStringForFiltering(opt.label).includes(formattedValue);`

Reply:
ouai t'as raison c'est overkill. Mais j'ai fait comme pour les directories. Je laisse comme ca pour que ce soit uniforme ou le fais avec `include`. Vous en pensez quoi ?

Reply:
avant c'était fait avec un includes, ça me semble pas incohérent de garder avec un includes